package.json

```diff
@@ -36,7 +36,7 @@
     "turbo": "latest"
   },
   "resolutions": {
-    "@balmy/sdk": "npm:@koaj/earn-sdk@0.0.17"
+    "@balmy/sdk": "npm:@koaj/earn-sdk@0.0.19"
```

Comment:
common should be updated too, and check if the root package.json isnt using the koaj one